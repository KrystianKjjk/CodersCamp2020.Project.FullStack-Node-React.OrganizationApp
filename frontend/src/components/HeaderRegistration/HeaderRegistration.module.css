@import url('https://fonts.googleapis.com/css2?family=Sarpanch:wght@600&display=swap');
@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@500&family=Sarpanch:wght@600&display=swap');

.header {
  width: 100%;
  padding: 10px 0;
  background-color: #292929;
  color: #ffffff;
  display: flex;
  align-items: center;
  justify-content: space-between;
  border-bottom: 1px solid #666666;
}

.logo {
  margin-left: 2rem;
  font-family: 'Sarpanch', sans-serif;
  font-size: 2rem;
}

.logo span {
  color: #ffc53d;
}
