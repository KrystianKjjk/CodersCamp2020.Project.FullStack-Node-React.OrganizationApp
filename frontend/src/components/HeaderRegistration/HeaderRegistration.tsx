import React from 'react'
import styles from './HeaderRegistration.module.css'

const HeaderRegistration: React.FC = () => {
  return (
    <div className={styles.header}>
      <div className={styles.logo}>
        <span>.</span>Coders<span>Camp</span>
      </div>
    </div>
  )
}

export default HeaderRegistration
