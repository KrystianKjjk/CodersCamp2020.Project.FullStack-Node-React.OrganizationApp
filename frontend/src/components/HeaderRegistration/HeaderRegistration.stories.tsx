import React from 'react';
import { Story, Meta } from '@storybook/react/types-6-0';
import HeaderRegistration from './HeaderRegistration';

export default {
  title: 'HeaderRegistration component',
  component: HeaderRegistration,
} as Meta;

const Template: Story = (args) => <HeaderRegistration {...args} />;

export const BasicHeaderRegistration = Template.bind({});
