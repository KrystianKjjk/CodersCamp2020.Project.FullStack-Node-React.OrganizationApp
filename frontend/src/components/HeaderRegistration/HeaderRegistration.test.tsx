import React from 'react'
import HeaderRegistration from './index'
import { render } from '@testing-library/react'

describe('HeaderRegistration', () => {
  it('should render header registration', () => {
    const { container } = render(<HeaderRegistration />)
    expect(container).toMatchSnapshot()
  })
})
