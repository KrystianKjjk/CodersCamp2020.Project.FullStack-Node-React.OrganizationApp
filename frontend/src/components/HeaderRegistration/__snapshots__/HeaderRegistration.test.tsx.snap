// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderRegistration should render header registration 1`] = `
<div>
  <div
    class="header"
  >
    <div
      class="logo"
    >
      <span>
        .
      </span>
      Coders
      <span>
        Camp
      </span>
    </div>
  </div>
</div>
`;
